README.md

```diff
@@ -363,9 +363,12 @@ Once your PR has been approved, it's time for your archive to make its debut!
 - Rerun the archiver without the `--sandbox` flag to create a draft production archive
 - Review the archive using the guidelines in Step 5.
 - Once published, submit the archive to the Catalyst Cooperative community.
-- Add the concept DOIs for the published sandbox and production to
+- Add the **concept DOIs** for the published sandbox and production to
 `/src/pudl_archiver/package_data/zenodo_doi.yaml`. These DOIs tell the archiver when a
 dataset already exists, making it possible to update existing archives with new data.
+This is the DOI listed after "Cite all versions? You can cite all versions by using the DOI"
+on the Zenodo webpage under the list of versions published. For a new archive,
+this should also be the record's DOI minus one.
```

Comment:
The last bit about minus 1 is a little confusing. I'm not sure if I'm supposed to subtract 1 from the value of the DOI (this seems wrong) or something else.

Reply:
I think the bit above directing people to the DOI listed after "cite all versions?...." is clear enough! If that's sufficient information, I think we can remove the bit about minus 1 to reduce potential confusion.